docs/pages/toolchain/katana/genesis.md

```diff
@@ -38,7 +38,11 @@ The genesis config file is a JSON file that contains the following fields:
     - `address` (optional)  
       _The fee token contract address._
   - `class` (optional)  
-    _The class of the fee token._
+    _The class of the fee token. It is has a value of either a classHash or a name._
+	- `classHash` (option)  
+      _The hash of the fee token class represented with a 0x- prefixed hex value._
```

Comment:
```suggestion
      _The hash of the fee token class hash (`0x` prefixed hex value)._
```

Reply:
A class hash is a hex value, so premising once that the class hash must be `0x` prefix is fine and then we can simply refer to it as `class hash`.